tests/performance/NetworkWrapper.h

```diff
@@ -22,14 +22,18 @@
 #include <memory>
 
 #include <olp/core/http/Network.h>
+#include <olp/core/porting/warning_disable.h>
 
 /*
  * Node test server is limited to http proxy. Wrapper alters ongoing requests
  * from https to http.
  */
 class Http2HttpNetworkWrapper : public olp::http::Network {
  public:
+  PORTING_PUSH_WARNINGS()
+  PORTING_CLANG_GCC_DISABLE_WARNING("-Wdeprecated-declarations")
   Http2HttpNetworkWrapper() : network_{olp::http::CreateDefaultNetwork(32)} {}
```

Comment:
Isn't it easier to use the new function? Something like `CreateDefaultNetwork({32, {}})` or provide a (empty?) ctor for `NetworkInitializationSettings`?